tests/e2e/test_forum.py

```diff
@@ -206,9 +208,16 @@ def test_basic_operations_thread(self):
         reply_content3 = "E2E sample reply 3 content E2E"
 
         self.init_and_enable_discussion()
+        self.ws = create_connection("ws://localhost:1501/ws")
+
         for upload_attachment in [False, True]:
             assert not self.thread_exists(title)
             attachment = self.create_thread(title, content, upload_attachment=upload_attachment)
+            assert self.thread_exists(title)
+
+            ws_msg = json.loads(self.ws.recv())
```

Comment:
this line is basically getting the socket message and converting it to a python dict in one step instead of:
```
ws_str_msg = self.ws.recv()
ws_dict_msg = json.loads(ws_str_msg)
```